src/apache/conf/custom.conf

```diff
@@ -0,0 +1,42 @@
+#####################################################################################
+# Use this file to override variables set in the default Apache configuration
+# WARNING: This could break or slow down your Nextcloud instance or make it
+# impossible for some clients to connect
+#
+# If you need to include a file stored in your writable partition,
+# use ${SNAP_DATA} as the prefix
+#####################################################################################
+
+#########################
+#        GLOBAL         #
+#########################
+#
+# LogLevel: Control the number of messages logged to the error_log.
+# Possible values include: debug, info, notice, warn, error, crit,
+# alert, emerg.
+#
+# LogLevel warn
+
+#########################
+#          SSL          #
+#########################
+# Warning the Mac OSX client currently requires TLSv1
+#SSLProtocol All -SSLv3 -TLSv1 -TLSv1.1
+#SSLCipherSuite ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA256
```

Comment:
I'm torn on this, because people plugging in a monitor and a keyboard and willing to modify their configs will find it easier to just uncomment what's there.
And I don't think the config is going change that often. Maybe twice a year and I suspect that people changing it will be people who keep track of these things.

What I'd like to introduce at some point is a way to push notifications from the Snap to Nextcloud. That way we can tell people about changes which have an impact on their custom configs, like if a major vulnerability in OpenSSL is published.